_sections/30-bindings.md

```diff
@@ -163,7 +163,7 @@ A subset of [XPath 1.0 functions](https://www.w3.org/TR/1999/REC-xpath-19991116/
 <a id="fn:current" href="#fn:current">`current()`</a> | As in [XForms 1.1](https://www.w3.org/TR/xforms11/#fn-current). Used inside predicates of expressions that use instance() to enable referring to a node relative to the context of the _current_ question. E.g. as in `instance('countries')/item[name=current()/../name]/capital`).
 <a id="fn:area" href="#fn:area">`area(node-set ns | geoshape gs)`</a> | Returns the calculated area in m2 of either a nodeset of geopoints or a geoshape value (not a combination of both) on Earth. It takes into account the circumference of the Earth around the Equator but does not take altitude into account.
 <a id="fn:distance" href="#fn:distance">`distance(node-set ns | geoshape gs | geotrace gt)`</a> | Returns the distance in meters of either a nodeset of geopoints or a single geoshape value or a single geotrace value (not a combination of these) on Earth, in the sequence provided by the points in the parameter. It takes into account the circumference of the Earth around the Equator and does not take altitude into account.
-<a id="fn:once" href="#fn:once">`once(* calc)`</a> | The parameter will be evaluated and returned if the context nodes's value is empty, otherwise the current value of the context node will be returned. The function is used e.g. to ensure that a random number is only generated once with `once(random())`.
+<a id="fn:once" href="#fn:once">`once(string calc)`</a> | The parameter will be evaluated and returned if the context nodes's value is empty, otherwise the current value of the context node will be returned. The function is used e.g. to ensure that a random number is only generated once with `once(random())`.
```

Comment:
`string` to me implies a string literal. Really this is like a [`PathExpr`](https://www.w3.org/TR/1999/REC-xpath-19991116/#NT-PathExpr) right? A literal or an XPath expression or a XPath path?

Reply:
I think this is correct. Wherever we've added string as the argument type, it can be each of those. It just indicates that the argument will be cast to a string (so the anything-goes type).

E.g. look at `if()` https://www.w3.org/TR/2003/REC-xforms-20031014/slice7.html#fn-if

Reply:
Ok, makes sense, thanks!